constellations/gemini-admin/src/session/sessionEdit.js

```diff
@@ -252,10 +253,17 @@ export class SessionEditPage extends React.Component {
                     />
                 </div>
 
-                <h4>Notes</h4>
-                <input
+                <InputText
+                    label="Notes"
+                    isTextBox={true}
```

Comment:
`description: "You may enter any additional information about this session here"`

Reply:
if the field is optional (not required), we don't need any validation